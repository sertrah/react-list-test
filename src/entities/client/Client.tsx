import React, { FunctionComponent } from "react"; // importing FunctionComponent

const Client: FunctionComponent= () => (
  <aside>
    <h2>Client</h2>
  </aside>
);

export default Client