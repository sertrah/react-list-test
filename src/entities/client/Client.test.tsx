import React from "react";
import ReactDOM from "react-dom";
import { render } from '@testing-library/react';
import Client from "./Client";

test("(<Client /> Matches snapshot", ()=> {
    const component = render(<Client />)
    expect(component.container).toMatchSnapshot();
})