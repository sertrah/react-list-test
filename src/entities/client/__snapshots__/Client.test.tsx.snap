// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(<Client /> Matches snapshot 1`] = `
<div>
  <aside>
    <h2>
      Client
    </h2>
  </aside>
</div>
`;
